### ISSUE TITLE
Attempting to Insert Mario while Path is set to an invalid file causes a segfault

### ISSUE THREAD

**Issue by KettLovahr** [libsm64/libsm64-blender#12]
Rather than just displaying an error, setting the Path to any file other than a valid z64 ROM will cause Blender to crash with a segmentation fault.

Tested on Arch Linux
Blender version 3.1.0
libsm64-blender version 1.0.5

It's just a minor inconvenience, nothing major, but failing gracefully would be preferable (and prevent potential loss of work!)

**Comment by jaburns** [comment]
Yeah this isn't ideal and it's an easy fix. I'll fix it for the next release, thanks for the report
